microprofile-fault-tolerance/src/main/java/org/wildfly/quickstarts/microprofile/faulttolerance/CoffeeResource.java

```diff
@@ -131,7 +135,7 @@ public Response availability(@PathParam("id") int id) {
 
     @GET
     @Path("/{id}/recommendations")
-    @Timeout(250)
+    @Timeout(value = 250)
```

Comment:
This change is unnecessary and also inconsistent with e.g. the annotation above it. The `value =` is implicit